pkgs/applications/misc/openbrf/default.nix

```diff
@@ -12,12 +12,22 @@ mkDerivation {
     sha256 = "16254cnr60ihcn7bki7wl1qm6gkvzb99cn66md1pnb7za8nvzf4j";
   };
 
-  buildInputs = [ qtbase vcg glew ];
+  patches = [
+    # split BoundaryWeight into BoundaryQuadricWeight and QualityQuadricWeight
+    # while preserving the original values where used.
+    # old: https://github.com/cnr-isti-vclab/vcglib/blob/a8e87662b63ee9f4ded5d4699b28d74183040803/vcg/complex/algorithms/local_optimization/tri_edge_collapse_quadric.h#L491
+    # new: https://github.com/cnr-isti-vclab/vcglib/blob/e4950d12e2db7f6ec3e587b06b2a0474b4f08d96/vcg/complex/algorithms/local_optimization/tri_edge_collapse_quadric.h#L590
+    ./vcgmesh.diff
+  ];
```

Comment:
Could you open an issue on the openbrf upstream and refer to it?

Reply:
Of course, how silly of me